spec/tbd_tests_spec.rb

```diff
@@ -1582,6 +1582,177 @@
   end
 end
 
+RSpec.describe TBD do
+  it "can process TB & D : DOE Prototype test_warehouse.osm + JSON I/O" do
+    translator = OpenStudio::OSVersion::VersionTranslator.new
+    path = OpenStudio::Path.new(File.dirname(__FILE__) + "/files/test_warehouse.osm")
+    os_model = translator.loadModel(path)
+    expect(os_model.empty?).to be(false)
+    os_model = os_model.get
+
```

Comment:
A TBD JSON output-as-input test. In a nutshell, if everything is spot on, the output file of a first run can be reused as the input file of a subsequent run (both output files should be identical).

Reply:
Nice!